HousingRepairsOnlineApi/Helpers/Constants.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System.Collections.Generic;
+
+namespace HousingRepairsOnlineApi.Helpers
+{
+    public static class Constants
```

Comment:
These constants are very specific to the types of confirmation that could be sent, please update the class name to reflect.
```suggestion
    public static class AppointmentConfirmationSendingTypes
```